dataset/dictionary/characters.json5

```diff
@@ -2854,24 +2860,29 @@
   {
     en: "Mehrak",
     ja: "メラック",
+    zhCN: "梅赫拉克",
     tags: [ "sumeru", "character-sub" ],
     notes: "カーヴェの工具箱",
   },
   {
     en: "Sorush",
     ja: "ソルシュ",
+    zhCN: "斯露莎",
     tags: [ "sumeru", "character-sub" ],
   },
   // TODO Because I'm not sure Apep / アペプ is the name of the new weekly boss,
   // I may update this word after v3.6 has released
+  // (new) アペプのオアシス守護者 is in Preview page of v3.6
   {
     en: "Apep",
     ja: "アペプ",
+    zhCN: "阿佩普",
     tags: [ "sumeru", "character-sub", "enemy-boss" ],
   },
   {
     en: "Dragon of Verdure",
     ja: "翠の草龍",
+    zhCN: "草龙(暂定)",   //TODO 中文版译名暂缺，“草龙”是俗称。
```

Comment:
Let's comment out this Chinese translation until the accurate traslation is revealed.

Reply:
Fixed in 810c233 and bb0a4d7.